sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusSenderAsyncClient.java

```diff
@@ -136,21 +137,23 @@ public String getEntityPath() {
     }
 
     /**
-     * Sends a message to a Service Bus queue or topic.
+     * Sends an array of messages to a Service Bus queue or topic using a batched approach. If the size of messages
+     * exceed the maximum size of a single batch, an exception will be triggered and the send will fail.
+     * By default, the message size is the max amount allowed on the link.
      *
-     * @param message Message to be sent to Service Bus queue or topic.
-     * @param sessionId the session id to associate with the message.
+     * @param messages Messages to be sent to Service Bus queue or topic.
      *
-     * @return A {@link Mono} the finishes this operation on service bus resource.
+     * @return The {@link Mono} the finishes this operation on service bus resource.
      *
-     * @throws NullPointerException if {@code message} or {@code sessionId} is {@code null}.
+     * @throws NullPointerException if {@code messages} is {@code null}.
```

Comment:
You have to indicate what exception is thrown when the payload exceeds the size of a single batch.